src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 public class ArtifactsSizeTest : SdkTests
 {
     private const int SizeThresholdPercentage = 25;
-    private static readonly string BaselineFilePath = BaselineHelper.GetBaselineFilePath($"ArtifactsSizes/{Config.TargetRid}.txt");
+    private static readonly string BaselineFilePath = BaselineHelper.GetBaselineFilePath($"{Config.TargetRid}.txt", "SdkContentTests");
```

Comment:
Why SdkContentTests vs ArtifactsSizeTests?  What is the logic to know when to use the test case name vs something else?

Reply:
ArtifactsSizeTests is under the SdkContentTests filter, so I lumped the artifacts baseline in with the other content test baselines. This made sense to me since the SDK pipeline runs all the sdk content tests and then publishes the results in a single directory.